packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -272,6 +276,14 @@ class DriveCommand extends RunCommandBase {
         );
       }
 
+      // Extract desiredCapabilities from file if specifying --web-desired-capablities-from-file
+      final String? desiredCapabilitiesJsonFilePath =
+          stringArg('web-desired-capabilities-from-file');
+      Map<String, dynamic>? desiredCapabilitiesJsonMap;
```

Comment:
Here and through this PR, avoid `dynamic` in favor of `Object?` https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

```suggestion
      Map<String, Object?>? desiredCapabilitiesJsonMap;
```